samples/Exporters/Console/TestConsoleExporter.cs

```diff
@@ -135,6 +135,11 @@ public override Task ShutdownAsync(CancellationToken cancellationToken)
             {
                 return this.next.ShutdownAsync(cancellationToken);
             }
+
+            public override Task ForceFlushAsync(CancellationToken cancellationToken)
```

Comment:
digression: Why the spec team chose `ForceFlush` instead of `Flush`? What is an unforced `Flush`? Oh well, let's stick to the spec.